.github/workflows/serverless.yml

```diff
@@ -115,10 +147,16 @@ jobs:
               MSSQL_SA_PASSWORD: "Localtestpass1!"
     env:
       PLUGINS: azure-functions
-      SERVICES: azureservicebusemulator,azuresqledge
+      SERVICES: azureservicebusemulator,azuresqledge,azurite,azureeventhubsemulator
     steps:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
-      - uses: ./.github/actions/node/active-lts
+      - run: |
+          docker cp \
```

Comment:
We need to add a config file for the Event Hubs emulator, but in github workflows we can't mount the file. The workaround is to user `docker cp` and restart the service. https://github.com/orgs/community/discussions/42127#discussioncomment-7591609